test/suite_test.go

```diff
@@ -32,22 +32,24 @@ import (
 )
 
 const (
-	kindVersion = "v0.8.1"
-	kindImage   = "kindest/node:v1.18.2"
-	testImage   = "securityoperators/seccomp-operator:latest"
+	kindVersion    = "v0.8.1"
+	kubectlVersion = "v1.18.5"
+	kindImage      = "kindest/node:v1.18.2"
+	testImage      = "securityoperators/seccomp-operator:latest"
 )
 
 type e2e struct {
 	suite.Suite
 	kindPath    string
+	kubectlPath string
 	clusterName string
 }
 
 func TestSuite(t *testing.T) {
 	suite.Run(t, &e2e{})
 }
 
-// SetupSuite downloads kind
+// SetupSuite downloads kind and kubectl
```

Comment:
In practice won't these be present in the kubekins image?

Reply:
Hm yeah, just saw that they're part of the image. Should I drop the download? I think this way we could decide on our own which version of kubectl we wanna use (probably always the latest)…

Reply:
It feels like it would be cleaner, do you know how any other projects handle this?

Reply:
Alright, I just changed to to lookup `kubectl` in `$PATH`.